ec/resources/waf/access_rule_test.go

```diff
@@ -85,3 +85,63 @@ func TestConvertInterfaceToAccessControls(t *testing.T) {
 		}
 	}
 }
+func TestFlattenAccessControls(t *testing.T) {
```

Comment:
Add a space between functions

Reply:
so spacing is being removed go formatter but just to confirm we should have function name func TestFlattenAccessControls (t *testing.T) {

Reply:
Hmm ok